WordPress/src/main/java/org/wordpress/android/ui/mysite/MySiteViewModel.kt

```diff
@@ -173,8 +173,11 @@ class MySiteViewModel @Inject constructor(
        as they're already built on site select. */
     private var isSiteSelected = false
 
+    private val isMySiteDashboardTabsFeatureConfigEnabled = mySiteDashboardTabsFeatureConfig.isEnabled()
+    private val isBloggingPromptsFeatureConfigEnabled = bloggingPromptsFeatureConfig.isEnabled()
```

Comment:
Nitpick: @ashiagr As you did for other config dependencies, `private val` can be removed from the below dependencies in the constructor of `MySiteViewModel`. 

``` 
private val mySiteDashboardTabsFeatureConfig: MySiteDashboardTabsFeatureConfig,
private val bloggingPromptsFeatureConfig: BloggingPromptsFeatureConfig,
```
to 
```
mySiteDashboardTabsFeatureConfig: MySiteDashboardTabsFeatureConfig,
bloggingPromptsFeatureConfig: BloggingPromptsFeatureConfig 
```

Reply:
Removed in ecaaf7c